### ISSUE TITLE
The admin charts aren't displayed

### ISSUE THREAD

**Issue by tmszi** [auto-mat/do-prace-na-kole#277]
**Describe the bug**
Admin charts aren't displayed.

**To Reproduce**

1. Go to https://dpnk.dopracenakole.cz/
2. Login into app
3.  Open web browser dev tools, switch to the Network tab (page)
4. Refresh page 
5. See error, js libs **d3**, **nvd3** required for [Django admin charts app](https://github.com/PetrDlouhy/django-admin-charts#configure-javascript-libraries) aren't loaded

![network_-_admin_charts_js_libs_not_found](https://user-images.githubusercontent.com/50632337/210494857-c30e0288-de75-4588-92e2-3af62d7bf53f.png)

**Expected behavior**
Admin charts should be displayed.

**Screenshots**
![admin_charts_aren_t_loaded](https://user-images.githubusercontent.com/50632337/210495156-02b07b02-33ee-4ad9-95b4-44890a9a27c6.png)

**Comment by PetrDlouhy** [comment]
@tmszi In the newest versions of `django-admin-charts` the JS files are taken from UNPKG. So maybe it will be enough to update the library and remove the configuration for bower: https://github.com/PetrDlouhy/django-admin-charts#configure-javascript-libraries

**Comment by tmszi** [comment]
> @tmszi In the newest versions of `django-admin-charts` the JS files are taken from UNPKG. So maybe it will be enough to update the library and remove the configuration for bower: https://github.com/PetrDlouhy/django-admin-charts#configure-javascript-libraries

Yes you are right. I readed `django-admin-charts` app  docs . But we can hold version "<1.2.0", because upgrade isn't smooth.
